vertx-opentracing/pom.xml

```diff
@@ -56,6 +48,46 @@
       <version>0.33.0</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-client</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>5.2.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.jaegertracing</groupId>
+      <artifactId>jaeger-core</artifactId>
+      <version>1.8.1</version>
+    </dependency>
+    <dependency>
```

Comment:
why is this dependency on itself needed ?

Reply:
This dependency is not needed, I've removed it. Thanks @vietj .